### PR TITLE
Update configure-server-endpoints.md

### PR DESCRIPTION
**Pull request by msbemba**
updated the heading to reduce ambiguity as the steps are used only when using installer scripts

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23615

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ You can use the [installer helper script](server-migration.md#installer-script)
 
 This script can be used in various scenarios, including those scenarios described in [Server migration scenarios from the previous, MMA-based Microsoft Defender for Endpoint solution](/microsoft-365/security/defender-endpoint/server-migration) and for deployment using Group Policy as described below.
 
-##### Apply the Microsoft Defender for Endpoint installation and onboarding packages using Group policy
+## Apply the Microsoft Defender for Endpoint installation and onboarding packages using Group policy when performing the installation with an installer script.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md]:
```suggestion
## Apply the Microsoft Defender for Endpoint installation and onboarding packages using Group policy when performing the installation with an installer script
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder.

**Pull request merged**